src/applications/edu-benefits/0993/containers/RoutedSavableSinglePageFormApp.jsx

```diff
@@ -192,9 +192,9 @@ class RoutedSavableApp extends React.Component {
       content = <LoadingIndicator message="Retrieving your profile information..."/>;
     } else {
       content = (
-        <FormApp formConfig={formConfig} currentLocation={currentLocation}>
+        <SinglePageFormApp formConfig={formConfig} currentLocation={currentLocation}>
           {children}
-        </FormApp>
+        </SinglePageFormApp>
```

Comment:
The only difference here is that SinglePageFormApp (which does not include the FormNav component) is used. Feedback on whether proposing that RoutedSavableFormApp should be updated to accept optional, alternative FormApp components, or whether proposing that FormApp should be updated to disable the FormNav is of interest or whether this use case is too rare to justify changing the common code is especially welcome.